changelog/910.feature.rst

```diff
@@ -0,0 +1 @@
+Add ``gif`` enumeration value to :class:`StickerFormatType`.
```

Comment:
```suggestion
Add :attr:`StickerFormatType.gif`.
```

Reply:
I didn't want to add something too short 🤷‍♂️